justfile

```diff
@@ -0,0 +1,284 @@
+# Configuration
+# set shell := ["bash", "-uc"]
+
+# Environment Variables
+export BASH_SILENCE_DEPRECATION_WARNING := "1"
+export CLICOLOR_FORCE := "1"  # For `tree`
+export LESS := "-FRXi"        # F: no-op single page, R: color, X: keep text when exiting, i: case insensitive
+export PACKAGES := "bind9-host curl fping git less tmux tree"
+
+# Get operating system
+OS := `uname -s`
+
+# Default recipe (shows available recipes)
+default:
+    @just --list
+
+# Development Environment Setup
+[private]
+_ensure-homebrew:
+    #!/usr/bin/env bash
+    if [[ "{{OS}}" == "Darwin" && ! -x $(command -v brew) ]]; then
+        [[ -d /opt/homebrew/bin ]] && export PATH="/opt/homebrew/bin:$PATH"
+        [[ -x $(command -v brew) ]] || echo "ERROR: homebrew missing"
+    fi
+
+# Activate virtual environment after changing directory
+a:
+    #!/usr/bin/env bash
+    echo $SHELL
+    check_conda=$(command -v conda)
+    echo $check_conda
+    [[ $(command -v conda) ]] && conda deactivate
+    might_be_file=$(command -v deactivate)
+    if [[ "${might_be_file}" ]]; then
+        if [[ -f "${might_be_file}" ]]; then
+            source deactivate
+        else
+            deactivate
+        fi
+    fi
+    if [[ -f .venv/bin/activate ]]; then
+        source .venv/bin/activate
+        just pathver python .python-version
+    elif [[ -d conda ]]; then
+        source "$HOME/miniconda3/etc/profile.d/conda.sh"
+        conda activate "$(basename "$PWD")"
+        just pathver python .python-version
+    fi
+    if [[ -f .nvmrc ]]; then
+        just pathver node .nvmrc
+    fi
+
+# print PATH and VERsion; optionally assert version file matches
+pathver command version_file="":
+    #!/usr/bin/env bash
+    source=$(type -p "{{command}}")
+    if [[ -z $source ]]; then
+        source=$(type "{{command}}")
+    fi
+    actual_version=$("{{command}}" --version | sed -E "s/({{command}} )?//i")
+    echo "$source $actual_version"
+    if [[ -f "{{version_file}}" ]]; then
+        expected_version=$(cat "{{version_file}}")
+        if [[ $actual_version != "$expected_version" ]]; then
+            echo "ERROR: $source version $actual_version does not match {{version_file}} $expected_version"
+            exit 1
+        fi
+    fi
+
+# Docker Compose Build
+dcb *ARGS:
+    docker compose --progress=plain build {{ARGS}}
```

Comment:
The justfile docs use lowercase https://github.com/casey/just?tab=readme-ov-file#quiet-recipes which I think I prefer
```suggestion
dcb *args:
    docker compose --progress=plain build {{args}}
```